This patch fixes an iOS bug where images aren't displayed, sourced from
the following commit:
https://github.com/facebook/react-native/commit/123423c

This file may be removed upon upgrading to react-native version 0.63.2
or greater.

Pivotal story for react-native 0.61 -> 0.63 upgrade:
https://www.pivotaltracker.com/story/show/175800398

diff --git a/node_modules/react-native/Libraries/Image/RCTUIImageViewAnimated.m b/node_modules/react-native/Libraries/Image/RCTUIImageViewAnimated.m
index 01aa75f..4ef8307 100644
--- a/node_modules/react-native/Libraries/Image/RCTUIImageViewAnimated.m
+++ b/node_modules/react-native/Libraries/Image/RCTUIImageViewAnimated.m
@@ -269,6 +269,8 @@ - (void)displayLayer:(CALayer *)layer
   if (_currentFrame) {
     layer.contentsScale = self.animatedImageScale;
     layer.contents = (__bridge id)_currentFrame.CGImage;
+  } else {
+    [super displayLayer:layer];
   }
 }
 
